### ISSUE TITLE
Does setting num_workers on a single GPU affect the convergence of the model?

### ISSUE THREAD

**Issue by Aiz-wallenstein** [zylo117/Yet-Another-EfficientDet-Pytorch#589]
Hello，thanks for your work! 
And  during the training I have this question : Does setting num_workers non_zero on a single GPU affect the convergence of the model? Can I have your opinion? Thanks very much!

**Comment by zylo117** [comment]
num_workers has nothing to do with it but the number of gpus does.
less gpus means smaller batchsize, which affects greatly on batchnorm layer, takes longer to converge and usually has worse performance.

**Comment by Aiz-wallenstein** [comment]
> num_workers has nothing to do with it but the number of gpus does.
> less gpus means smaller batchsize, which affects greatly on batchnorm layer, takes longer to converge and usually has worse performance.

Thanks! I get it!
